### PR TITLE
Fix reading  of boolean env variable

### PR DESCRIPTION
**Pull request by ewelinajozwik**

### CODE REVIEW

File: dicer/main.py [hunk 1]
```diff
@@ -51,7 +51,7 @@ def read_config() -> TransmartConfiguration:
     config = TransmartConfiguration(
         url=read_env_variable('TRANSMART_URL'),
         keycloak_config=keycloak_config,
-        verify_cert=os.environ.get('VERIFY_CERT', True)
+        verify_cert=os.environ.get('VERIFY_CERT', 'True').lower() != 'false'
```
Comment (gijskant) [inline dicer/main.py]: I thought that `verify_cert` should be of type `Union[str, bool]` to also allow usage like `requests.get('https://github.com', verify='/path/to/certfile')`...
So then we need more elaborate parsing of this value:
```python
True if value is None or value == '' or value.lower() == 'true'
     else False if value.lower() == 'false' else value
```

Comment (gijskant) [reply dicer/main.py]: Maybe raise exception if the path does not exist.

Comment (gijskant) [reply dicer/main.py]: Actually, I now see that 

> This list of trusted CAs can also be specified through the REQUESTS_CA_BUNDLE environment variable.

(https://2.python-requests.org/en/master/user/advanced/)

Comment (ewelinajozwik) [reply dicer/main.py]: right... I forgot about the path option. The problem is then that: 

> Self-signed SSL certificates specified in REQUESTS_CA_BUNDLE will not be taken into account. As a result an SSL: CERTIFICATE_VERIFY_FAILED is thrown.

**Comment by gijskant** [comment]
Another, related issue is that the `KeycloakRestClient` if not configured with the verify option. Is that intentional?

**Comment by ewelinajozwik** [comment]
@gijskant KeycloakRestClient and verify_cert type are updated now

**Review by gijskant — APPROVED** [review]
Nice!

**Pull request merged**